dominion/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require("path")
+const webpack = require('webpack')
+//const BundleTracker = require('webpack-bundle-tracker')
+//const HWP = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, '/dominion_ui/src/index'), // entry point of our app. assets/js/index.js should require other js modules and dependencies it needs
+
+  output: {
+      path: path.resolve('./static/bundles/'),
+      filename: "main.js",
+  },
+
+  watch: true,
+
+  plugins: [
+  //  new BundleTracker({filename: './webpack-stats.json'}),
+  //  new HWP({template: path.join(__dirname,'/dominion/templates/homepage.html')})
```

Comment:
Minor nitpick, if we don't need it maybe just remove them all together.
Even from `package.json`.